src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/GetElevationAtAPoint/readme.md

```diff
@@ -0,0 +1,37 @@
+# Get elevation at a point
+
+Get the elevation for a given point on a surface in a scene. 
+
+![](GetElevationAtAPoint.jpg)
+
+## Use case
+
+Knowing the elevation at a given point in a landscape can aid in navigation, planning and survey in the field.
+
+## How to use the sample
+
+Tap anywhere on the surface to get the elevation at that point. Elevation is reported in meters since the scene view is in WGS84.
+
+## How it works
+
+1. Create a SceneView and Scene with an imagery base map.
+2. Set an ArcGISTiledElevationService as the elevation source of the scene's base surface.
+3. Use the screenToBaseSurface(screenPoint) method on the scene view to convert the tapped screen point into a point on surface.
```

Comment:
Don't forget to review the readme for code changes. It looks like in this case we're using `surface.GetElevationAsync`. It looks like the screenToBaseSurface step may not be needed either. Don't forget **`** for code.